src/app/services/baw-api/security.service.ts

```diff
@@ -23,35 +23,34 @@ export class SecurityService extends BawApiService {
 
     this.loggedInTrigger.next(this.isLoggedIn());
     this.paths = {
-      register: "/security",
-      signIn: "/security",
-      signOut: "/security"
+      register: this.makeTemplate`/security`,
```

Comment:
you probably don't _need_ to use make template around a string with no parameters

Reply:
This is mostly just for standardising the services before i tackle #78 .